packages/composites/anchor/src/endpoint/price/index.ts

```diff
@@ -14,63 +14,85 @@ import * as bluna from './bluna'
 export const supportedEndpoints = ['price']
 
 export const inputParameters: InputParameters = {
-  from: ['base', 'from', 'coin'],
-  to: ['quote', 'to', 'market'],
-  quoteDecimals: false,
-  source: false,
-  terraBLunaContractAddress: false,
+  from: {
+    required: true,
+    aliases: ['base', 'from', 'coin'],
+    description: 'The symbol of the currency to query',
+    options: ['bETH', 'bLUNA'],
+  },
+  to: {
+    required: true,
+    aliases: ['quote', 'to', 'market'],
+    description: 'The symbol of the currency to convert to',
+    options: ['BTC', 'ETH', 'USD'],
```

Comment:
Just to be clear, this means the feed can only use these 3 assets for the quote asset. We had a few other input params in the past where the README only listed a few as options but they were meant as examples for a larger set...

Reply:
yup it'll only work with these 3 assets.